specs/lightbeam.sh

```diff
@@ -12,7 +12,7 @@ while(true); do
     if [[ $? -ne 0 ]]; then FAIL=1; fi
     status_code=`curl -s -o /dev/null -w "%{http_code}" http://localhost/api/health`
     echo "Lightbeam cluster health check: $status_code"
-    if [[ $? -ne 0 || $status_code -ne 200 ]]; then FAIL=1; fi
+    if [[ $? -ne 0 || $status_code -ne 200 || $status_code -ne 301 ]]; then FAIL=1; fi
```

Comment:
This needs to be an and not an or. So something like below, you will have to test out the change.
```suggestion
    if [[ $? -ne 0 || ($status_code -ne 200 && $status_code -ne 301) ]]; then FAIL=1; fi
```

Reply:
You are right, the service kept getting restarted. I have modified this.